src/Table/TableCellText.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import css from 'styled-jsx/css'
+
+import { Consumer } from './tableContext'
+
+const tableCellTextStyles = css`
+    span {
+        display: inline-block;
+        padding: 9px 0;
+        font-size: 14px;
+        line-height: 18px;
+    }
+
+    :global(tbody) span {
+        padding: 13px 0;
+    }
+`
+
+const tableCellTextStylesResponsive = css`
+    @media (max-width: 400px) {
+        :global(tbody) span {
+            padding: 3px 0;
+        }
+    }
+`
+
+const TableCellTextStatic = ({ label }) => (
+    <span>
+        {label}
+        <style jsx>{tableCellTextStyles}</style>
+    </span>
+)
+
+const TableCellTextResponsive = ({ label }) => (
+    <span>
+        {label}
+        <style jsx>{tableCellTextStyles}</style>
+        <style jsx>{tableCellTextStylesResponsive}</style>
+    </span>
+)
+
+export const TableCellText = ({ label }) => (
```

Comment:
Is there ever a case where `TableCell` might be used without `TableCellText`?  If not, it would be nice to remove the boilerplate nexting of a `TableCellText` within every `TableCell`.  If there might be cases where it isn't used, maybe that could be encapsulated in props to the `TableCell`?

Additionally, wouldn't you use this same wrapper for non-text content like an image?  If we keep the explicit child it should probably be called `TableCellContent` or something more generic.

Reply:
I see your opening comment now, but my question still stands.  I think I would prefer a singular `TableCell` component with an optional `noPadding={true}` prop or something to skip the defaults (which should be default).

Reply:
> Is there ever a case where TableCell might be used without TableCellText?

Yes, it can be buttons, icons, `Chip`s, etc.

> Additionally, wouldn't you use this same wrapper for non-text content like an image? If we keep the explicit child it should probably be called TableCellContent or something more generic.

The component contains mostly css rules for text. I supposed it'd work for images (and other inline-block elements) as well.

I'll play around a bit and try to figure out if this can be incorporated into the table cell component